Refactor landing page and add pages for each usecase. (#151)
VirtusLab/scala-cli: README.md

from Maven Central upon first launch (using the coursier cache, just like
a coursier bootstrap).

### Website 

> Website uses [Docusaurus](https://v1.docusaurus.io/en/) and under the hood it uses [infima](https://infima.dev/docs/layout/spacing) framework to style the website. 

Reviewer: ?

```suggestion
The Scala CLI website is built with [Docusaurus](https://v1.docusaurus.io/en/) and uses [Infima](https://infima.dev/docs/layout/spacing) for styling. 
```
